agent/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM alpine:3.7
+
+RUN apk add --update git curl bash openssh
+
+# Install `kubectl`
+RUN curl -sLO https://storage.googleapis.com/kubernetes-release/release/v1.9.3/bin/linux/amd64/kubectl &&\
```

Comment:
In this case, we don't need the AN wrapper because we don't have to worry about the kerberos auth stuff?